### ISSUE TITLE
Memory Leak w/ notdhcpserver

### ISSUE THREAD

**Issue by max-b** [sudomesh/notdhcpserver#9]
This is from treeherder's mesh node: 
100.64.20.193
![treeherder_memory_graph_image](https://cloud.githubusercontent.com/assets/2491841/11055412/ef663af8-872b-11e5-97c8-4211b6e5afa4.png)

```
Mem: 107176K used, 18980K free, 820K shrd, 3040K buff, 8776K cached
CPU:   1% usr   1% sys   0% nic  96% idle   0% io   0% irq   0% sirq
Load average: 0.02 0.06 0.05 1/47 29110
  PID  PPID USER     STAT   VSZ %VSZ %CPU COMMAND
28498     2 root     SW       0   0%   1% [kworker/u2:1]
 1846     1 root     S     1636   1%   1% /usr/sbin/hostapd -P /var/run/wifi-ph
 1842     1 root     S     1632   1%   0% /usr/sbin/hostapd -P /var/run/wifi-ph
 2737     1 root     S    75220  60%   0% /usr/bin/notdhcpserver -vf -s /opt/no
29072 28943 root     R     1364   1%   0% top
```

I don't have time atm to go and debug, but wanted to flag it here to make sure we didn't miss/forget it.

**Comment by max-b** [comment]
It looks like it's incrementing about 4 bytes every 30 seconds or so....
ca862c358e17954887c379505b1fe7a076c51a79 doesn't seem to have fixed it, and 1f4d836ea88f51e90e5ae036c8849804acb7711a didn't fix the bug, so I had to push 86b3ab8 which reverted the code changes in swlib.

**Comment by max-b** [comment]
1d0d84b8fb1b590a5b6fb6df02bd5332ae07e486 and dff1244c74186b07eb5f4c2f6ad1c5975564df34 fix this. Thanks @papazoga!
